src/NLU.DevOps.ModelPerformance/NLUAccuracy.cs

```diff
@@ -14,128 +15,182 @@ namespace NLU.DevOps.ModelPerformance
+            var allIntentsF1 = Print(compareResults.Statistics.Intent.F1(), baseline?.Intent.F1());
+            var allIntentsTotal = Print(compareResults.Statistics.Intent.Total(), baseline?.Intent.Total(), 0);
+            var allIntentsFP = Print(compareResults.Statistics.Intent.FalsePositive, baseline?.Intent.FalsePositive, 0);
+
+            Console.WriteLine("## Intents Results");
+
+            var intentTable = new ConsoleTable("Intent", "Precision", "Recall", "F1", "Total", "FP");
+            intentTable.AddRow("*", allIntentsPrecision, allIntentsRecall, allIntentsF1, allIntentsTotal, allIntentsFP);
+
+            compareResults.Statistics.ByIntent
+                .OrderBy(intentItem => intentItem.Key)
+                .ToList()
+                .ForEach(intentItem =>
+                {
+                    var baselineResults = default(ConfusionMatrix);
+                    if (baseline != null && !baseline.ByIntent.TryGetValue(intentItem.Key, out baselineResults))
+                    {
+                        baselineResults = ConfusionMatrix.Default;
+                    }
+
+                    var intentPrecision = Print(intentItem.Value.Precision(), baselineResults?.Precision());
+                    var intentRecall = Print(intentItem.Value.Recall(), baselineResults?.Recall());
+                    var intentF1 = Print(intentItem.Value.F1(), baselineResults?.F1());
+                    var intentTotal = Print(intentItem.Value.Total(), baselineResults?.Total(), 0);
+                    var intentFP = Print(intentItem.Value.FalsePositive, baselineResults?.FalsePositive, 0);
+                    intentTable.AddRow(intentItem.Key, intentPrecision, intentRecall, intentF1, intentTotal, intentFP);
+                });
+
+            intentTable.Write(Format.MarkDown);
+
+            var allEntitiesPrecision = Print(compareResults.Statistics.Entity.Precision(), baseline?.Entity.Precision());
+            var allEntitiesRecall = Print(compareResults.Statistics.Entity.Recall(), baseline?.Entity.Recall());
+            var allEntitiesF1 = Print(compareResults.Statistics.Entity.F1(), baseline?.Entity.F1());
+            var allEntitiesTotal = Print(compareResults.Statistics.Entity.Total(), baseline?.Entity.Total(), 0);
+            var allEntitiesFP = Print(compareResults.Statistics.Entity.FalsePositive, baseline?.Entity.FalsePositive, 0);
+
+            Console.WriteLine("## Entity Results");
+
+            var entityTable = new ConsoleTable("Entity", "Precision", "Recall", "F1", "Total", "FP");
+            entityTable.AddRow("*", allEntitiesPrecision, allEntitiesRecall, allEntitiesF1, allEntitiesTotal, allEntitiesFP);
+
+            compareResults.Statistics.ByEntityType
+                .OrderBy(entityItem => entityItem.Key)
+                .ToList()
+                .ForEach(entityItem =>
+                {
+                    var baselineResults = default(ConfusionMatrix);
```

Comment:
>baselineResults [](start = 24, length = 15)

and this can be entityBaselineResults or entityBaseline makes reading the code easier.